### PR TITLE
2022q2: add lldb project info

### PR DESCRIPTION
**Pull request by mgorny**
CC @emaste

### CODE REVIEW

File: 2022q2/lldb.adoc [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+Contact: Kamil Rytarowski <kamil@moritz.systems> +
+Contact: Michał Górny <mgorny@moritz.systems>
+
+According to the upstream description, "LLDB is a next generation,
+high-performance debugger. It is built as a set of reusable components which
+highly leverage existing libraries in the larger LLVM Project, such as the
+Clang expression parser and LLVM disassembler."
+
+FreeBSD includes LLDB in the base system.  At present, it has some limitations
+compared to the GNU GDB debugger, and does not yet provide a complete
```
Comment (emaste) [inline 2022q2/lldb.adoc:16]: I wonder if we should soften this statement somewhat -- your previous work brought lldb to be a credible replacement to the GDB version that was in the base system.

Maybe something like "Previous sponsored projects improved LLDB, to make it a credible debugger for the base system, although it still has a few limitations compared to contemporary versions of GNU GDB."

Comment (mgorny) [reply 2022q2/lldb.adoc:16]: Thanks, that's a very good suggestion. I'll update the text shortly.

File: 2022q2/lldb.adoc [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+for multiprocess debugging.  The client featured support for debugging multiple
+independent processes simultaneously via maintaining multiple connections
+to different server instances.  Thanks to our earlier work, the server was
+able to process `fork(2)` and `vfork(2)` calls and either detach the newly
+forked child and continue tracing the parent process, or detach the parent
+and follow the child (equivalent to GDB's `follow-fork-mode` setting).
+
+Once the project is finished, LLDB will be able to trace an arbitrary number
+of forked processes simultaneously (equivalent to GDB's `detach-on-fork off`).
+A full support for the multiprocess extension to the GDB Remote Serial Protocol
```
Comment (pauamma) [inline 2022q2/lldb.adoc:31]: s/A full support/Full support/

**Review by pauamma — APPROVED** [review]
Other than that, LGTM. I can fix that nit when I merge the first batch.

File: 2022q2/lldb.adoc (hunk 2 above)
Comment (mgorny) [reply 2022q2/lldb.adoc]: Thanks. This is practically zero effort for me, so I've just updated it myself.

**Review by pauamma — APPROVED** [review]

**Comment by mgorny** [comment]
Thanks!

**Pull request merged**